docs/maintaining.md

```diff
@@ -160,22 +190,39 @@ curl -X PUT --header 'Content-Type: application/json' -d '
 
 ### Resetting your GraphDB instance
 
-If you previously set up a Neurobagel node on your machine but want to reset your graph database to start again _from scratch_, 
-the most foolproof way would be to start with a clean GraphDB configuration to avoid conflicts with any previously created credentials or databases.
+Each Neurobagel node has its own GraphDB instance, which is used to store the graph data for the node.
+If you want to reset your graph database and start again from scratch,
+follow these steps:
+
+1. Ensure that your Neurobagel node is not running (i.e., shut down the Docker containers for the node).
+
+    ```bash
+    docker compose --profile full_stack down
+    ```
+
+    If you are not using the `full_stack` profile, replace `full_stack` with the name of the profile you are using.
+
+2. Delete the docker volume that contains the GraphDB data for your node.
+
+    ```bash
+    docker volume rm neurobagel_graphdb_data
+    ```
+
+    replace `neurobagel_graphdb_data` with the name of the volume that was created for your node.
+    It is usually named `<project_name>_graphdb_data`
+    where `<project_name>` is the name of the directory where you have stored the `docker-compose.yml` file for your Neurobagel node.
+
+3. Start your Neurobagel node again.
+
+    ```bash
+    docker compose --profile full_stack up -d
+    ```
 
 Some examples of when you might want to do this:
 
 - You started but did not complete Neurobagel node setup previously and want to ensure you are using up-to-date instructions and recommended configuration options
 - Your local node has stopped working after a configuration change to your graph database (e.g., your Neurobagel node API no longer starts or responds with an error, but you have confirmed all environment variables you have set should be correct)
 
-The configuration for a given GraphDB instance is not tied to a specific GraphDB Docker container, but to the persistent home directory for GraphDB on the host machine.
-
-So, to 'reset' your GraphDB instance for Neurobagel, you need to clear the contents of your persistent GraphDB home directory on your filesystem (this is the path specified for `NB_GRAPH_ROOT_HOST` in your `.env`, which is `~/graphdb-home` by default).
-
 !!! warning
```

Comment:
given the suggested info box above, I think we can remove this! (probably not super useful to be at the very end anyways)